### PR TITLE
Update test wrapper to match new k8s

### PR DESCRIPTION
**Pull request by soltysh**
This is to help land https://github.com/openshift/origin/pull/27570

/assign @ingvagabund

### CODE REVIEW

File: openshift-hack/e2e/annotate/rules.go [hunk 1]
```diff
@@ -34,7 +34,6 @@ var (
 			`\[Feature:GPUDevicePlugin\]`,
 			`\[sig-scheduling\] GPUDevicePluginAcrossRecreate \[Feature:Recreate\]`,
 
-			`\[Feature:ImageQuota\]`,                    // Quota isn't turned on by default, we should do that and then reenable these tests
```
Comment (ingvagabund) [inline openshift-hack/e2e/annotate/rules.go:37]: Has the quota been turned on by default?

Comment (soltysh) [reply openshift-hack/e2e/annotate/rules.go:37]: No, this is an origin test, see https://github.com/openshift/origin/blob/cd282f24b7e94261b0bfaf51fa6265eb6e1799a6/test/extended/imageapis/quota_admission.go#L31 and https://github.com/openshift/origin/blob/cd282f24b7e94261b0bfaf51fa6265eb6e1799a6/test/extended/imageapis/quota_admission.go#L31 so it should live there, not in o/k.

The general rule is that o/k annotation rules live in this repo, whereas o/o rules in origin.

**Comment by soltysh** [comment]
Manually adding valid-bug, this is only updating test helper needed to land https://github.com/openshift/origin/pull/27570, so it's non-disruptive.

**Pull request merged**